File: tests/fixtures/symbol_index_project/java_pkg/Service.java
package java_pkg;

import java.util.List;
import java_pkg.Helper;

public class Service {
    private Helper helper;

    public Service() {
        this.helper = new Helper();
    }

    public boolean run(String value) {
        Helper.staticName();
        return helper.validate(value);
    }
}

File: tests/fixtures/symbol_index_project/python_pkg/service.py
import os

import python_pkg.utils as utils_alias

from .utils import Helper
from .utils import load_config as load_settings


class Service:
    def __init__(self):
        self.helper = Helper()

    def validate(self, value: str) -> bool:
        return self.helper.validate(value)

    def run(self) -> dict[str, str]:
        os.getcwd()
        load_settings("app.toml")
        utils_alias.load_config("app.toml")
        method_name = "validate"
        getattr(self.helper, method_name)(value="x")
        return load_settings("app.toml")
